warehouse/manage/forms.py

```diff
@@ -382,6 +382,16 @@ def validate_macaroon_id(self, field):
 # /manage/organizations/ forms
 
 
+class InformationRequestResponseForm(wtforms.Form):
+    response = wtforms.TextAreaField(
+        validators=[
+            wtforms.validators.InputRequired(
+                message="Provide your response to the request."
+            )
```

Comment:
Curious - do we only localize the validation error messages, or should we localize placeholder messages as well? It's sometimes yes/sometimes no in this module.

Reply:
Hm, just looks like inconsistencies. I'll set this string as translatable though. Probably worth filing an issue to have someone audit more fully at some point in time.

Reply:
96aacc53b52d16151c2c9564714ef3350904b292